pagopa/io-app
# feat(IT Wallet): [SIW-1918] Add iPatente CTA in MDL details screen

## Short description
This PR adds the iPatente service CTA in the MDL details screen

## List of changes proposed in this pull request
- Bumped `IO_SERVICES_METADATA_VERSION` to version `1.0.52`
- Moved ITW remote config selectors to `features/itWallet/common/store/selectors/remoteConfig.ts`
- Added `itwIsIPatenteCtaEnabledSelector` selector
- Added iPatente CTA in the MDL details screen

## How to test
Navigate to the MDL details screen, check that the iPatente CTA is visible.
Mock the `/status/backend` response setting itw.`ipatente_cta_enabled` to `false` and check that the CTA is not visibile.
